uxarray/grid/coordinates.py

```diff
@@ -1,15 +1,14 @@
 import xarray as xr
 import numpy as np
-
+import math
 import warnings
 
-from uxarray.constants import ERROR_TOLERANCE
 from uxarray.conventions import ugrid
-
-from typing import Union
+from uxarray.grid.arcs import _angle_of_2_vectors
```

Comment:
This appears to be the root cause of the circular import.

To avoid this, can you try using the following in this module when calling this function:

`uxarray.grid.arcs._angle_of_2_vectors`

Reply:
> This appears to be the root cause of the circular import.
> 
> To avoid this, can you try using the following in this module when calling this function:
> 
> `uxarray.grid.arcs._angle_of_2_vectors`

Doesn't work. Numba doesn't like uxarray...